libcloudforensics/providers/gcp/forensics.py

```diff
@@ -124,44 +125,52 @@ def StartAnalysisVm(
     project: str,
     vm_name: str,
     zone: str,
-    boot_disk_size: int,
-    boot_disk_type: str,
-    cpu_cores: int,
+    boot_disk_size: int = 10,
+    boot_disk_type: str = 'pd-standard',
+    cpu_cores: int = 4,
```

Comment:
it actually works if you don;t pass a keyword argument before then it can be positional, so this works: 
```
forensics.StartAnalysisVm(
     "project",
     "forensics-instance",
     "us-central1-b", 10, 'pd-ssd', 4,
     attach_disks=['disk'])
```
and this works:
```
python3 tools/cli.py gcp --project project-id startvm testing-vm2 us-central1-c --attach_disks disk-name --disk_type=pd-standard --disk_size=10 --cpu_cores=4
```